app/models/user.rb

```diff
@@ -0,0 +1,19 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
+
+  has_many :items
+  has_many :orders
```

Comment:
```suggestion
  has_many :items
  has_many :orders
```
アソシエーションを削除しましょう。

■理由
ユーザー管理機能の時点ではユーザーモデルしか作成せず、他モデルとのアソシエーションが不要になるためです。